doc/README.coding-style

```diff
@@ -147,20 +145,23 @@ This example is *bad*:
 	s = sizeof( struct file );
 
 3.2.3 When declaring pointer, the preferred use of '*' is adjacent to the data
-name or function name. E.g.:
+      name or function name. E.g.:
 
 	char *linux_banner;
 	unsigned long long memparse(char *ptr, char **retptr);
 	char *match_strdup(substring_t *s);
 
-3.2.4 When type conversin, add a space before '*'.
+3.2.4 When type casting, a space before '*' and/or after the right paren is
+      allowed but not imperative.  E.g:
+
+	byte_len = strlen((char *) utf8_string);
 
-E.g:
+        - or -
 
-	hostSalt = (cl_uint *) mem_alloc(SALT_BUFFER_SIZE);
+	byte_len = strlen((char*)utf8_string);
```

Comment:
For new code, I'd rather have us standardize on `(char *)utf8_string` - that is, space before the `*)`, no space after. This is consistent with e.g. `char *string;`.

Reply:
I'm fine with that. My habit of dropping the other space is a remnant from the 80-column days